### PR TITLE
mcobbett bump framework version

### PR DESCRIPTION
**Pull request by techcobweb**
- bump version number of framework

### CODE REVIEW

File: galasa-extensions-parent/buildSrc/src/main/groovy/galasa.extensions.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 dependencies {
-    implementation 'dev.galasa:dev.galasa.framework:0.22.0'
+    implementation 'dev.galasa:dev.galasa.framework:0.27.0'
     implementation 'dev.galasa:dev.galasa:0.21.0'
```
Comment (techcobweb) [inline galasa-extensions-parent/buildSrc/src/main/groovy/galasa.extensions.gradle:8]: Should this be 0.27.0 ? Seems likely ?

Comment (techcobweb) [inline galasa-extensions-parent/buildSrc/src/main/groovy/galasa.extensions.gradle:8]: Should this be 0.27.0 ? Seems likely ?

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Review by eamansour — APPROVED** [review]

File: galasa-extensions-parent/buildSrc/src/main/groovy/galasa.extensions.gradle (hunk 1 above)
Comment (eamansour) [reply galasa-extensions-parent/buildSrc/src/main/groovy/galasa.extensions.gradle:8]: The latest version of dev.galasa is 0.21.0 https://mvnrepository.com/artifact/dev.galasa/dev.galasa

Not sure why it hasn't kept up with the releases.

**Pull request merged**